app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.kt

```diff
@@ -82,7 +82,8 @@ fun setImageDrawable(imageView: ImageView, thumbnailGraphic: SkillThumbnailGraph
  * @param colorOrUri Represents either a colorId or local image uri.
  */
 @BindingAdapter("profile:src")
-fun setProfileImage(imageView: ImageView, profileAvatar: ProfileAvatar) {
+fun setProfileImage(imageView: ImageView, profileAvatar: ProfileAvatar?) {
+  if (profileAvatar == null) return
```

Comment:
What would the imageview show in this case?

Reply:
This case is needed because initially the binding is called with a null ProfileAvatar, but it gets called again right afterwards with the correct ProfileAvatar. We don't need to show anything initially.